.github/workflows/ci.yml

```diff
@@ -211,7 +211,7 @@ jobs:
         with:
           tool: nextest
       - name: Run tests with Miri
-        run: cargo miri nextest run --no-fail-fast
+        run: cargo miri nextest run --no-fail-fast --workspace --exclude vortex-file --exclude vortex-layout --exclude vortex-sampling-compressor --exclude vortex-fsst
```

Comment:
vortex-fsst got an axe since fsst crate independently runs miri and it has a very slow test. Sampling-compressor is not useful if every encoding is tested independetly but it has some lengthy compression tests